tools/update_python_pip_packages.rb

```diff
@@ -8,10 +10,20 @@
 py3_files = Dir['py3_*.rb']
 pip_files = `grep -l "^require 'buildsystems/pip'" *.rb`.chomp.split
 relevant_pip_files = (py3_files + pip_files).uniq!
+
+pip_config = `pip config list`.chomp
+system 'pip config --user set global.index-url https://gitlab.com/api/v4/projects/26210301/packages/pypi/simple', %i[err out] => File::NULL unless pip_config.include?("global.index-url='https://gitlab.com/api/v4/projects/26210301/packages/pypi/simple'")
```

Comment:
We have this `https://gitlab.com/api/v4/projects/26210301/packages` url hardcoded in several places throughout the repo.  Would it make sense to add a constant?

Reply:
Does `CREW_GITLAB_PKG_REPO` work?

Reply:
That's what I did...